servicetalk-http-netty/src/main/java/io/servicetalk/http/netty/DefaultSingleAddressHttpClientBuilder.java

```diff
@@ -241,8 +242,16 @@ public StreamingHttpClient buildStreaming() {
     }
 
     private static <U, R> StreamingHttpClient buildStreaming(final HttpClientBuildContext<U, R> ctx) {
-        final ReadOnlyHttpClientConfig roConfig = ctx.httpConfig().asReadOnly();
         final HttpExecutionContext executionContext = ctx.builder.executionContextBuilder.build();
+        // XXX This modifies the builder state. Suggestions for allowing execution strategy to override but not
```

Comment:
Wouldn't noOffloadsStrategy dictate that the `asyncCloseOffload` setting is ineffective?

Reply:
yes, it was confusing. Hopefully better now with the new approach